manager_install.sh

```
@@ -0,0 +1,8 @@
+#! /bin/bash
+
+cd evolution-manager-v2
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)